packages/components/src/VirtualizedList/VirtualizedList.test.js

```diff
@@ -43,6 +43,34 @@ describe('VirtualizedList', () => {
 		expect(wrapper.node).toMatchSnapshot();
 	});
 
+	it('should use AutoSizer with defaultHeight', () => {
+		// when
+		const wrapper = shallow(
+			<VirtualizedList
+				collection={collection}
+				defaultHeight={600}
+				id={'my-list-id'}
+				isSelected={jest.fn()}
+				rowHeight={50}
+				selectionToggle={jest.fn()}
+				sort={jest.fn()}
+				sortBy={'name'}
+				sortDirection={'DESC'}
+				type={TABLE}
+			>
+				{contentFields}
+			</VirtualizedList>,
+		);
+
+		const renderer = wrapper.node.props.children;
+
+		// when
+		const rendererInstance = renderer({ height: 0, width: 300 });
+
+		// then
+		expect(rendererInstance).toMatchSnapshot();
```

Comment:
I don't understand this tests. sorry. What do you really expect ?

Reply:
I'm expecting `height: <DEFAULT_HEIGHT>` if AutoSizer returns `height: 0`

Reply:
In that case please remove snapshots and do
```javascript
expect(rendererInstance.props().height).toBe(600)
```